checker/manual/creating-a-checker.tex

```diff
@@ -37,7 +36,7 @@ \chapter{How to create a new checker\label{writing-a-checker}}
 checker.  Specific checkers that are designed for extension (besides the Subtyping
 Checker) include the Fake Enumeration Checker
 (\chapterpageref{fenum-checker}), the Units Checker
-(\chapterpageref{units-checker}), and the typestate checkers
+(\chapterpageref{units-checker}), and the Typestate Checker
```

Comment:
I agree with the change to singular, but I had undone the capitalization because I see no evidence that the author uses the proper name "Typestate Checker" for his implementation".